### PR TITLE
Bounty Policy

### PR DESCRIPTION
**Pull request by znarf**
See: https://github.com/opencollective/opencollective-frontend/blob/bounty-md/BOUNTY.md

### CODE REVIEW

File: BOUNTY.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+   - If you can issue an invoice, submit it as expense to [Open Collective Engineering](https://opencollective.com/engineering).
+   - If you can't issue an invoice, ask to be rewarded with an Open Collective Gift Card.
+
+### Workflow for Core Contributors
+
+1. Tag issues with the "bounty" label and another tag with the bounty amount (ie: "100\$")
+
+1. Add a comment with basic explanations of the Bounty process and a link to BOUNTY.md
+
+1. Make sure the issue is understandable for newcoeers and the expectations are clearly set
```
Comment (Betree) [inline BOUNTY.md]:
```suggestion
1. Make sure the issue is understandable for newcomers and the expectations are clearly set
```

File: BOUNTY.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+1. Get paid:
+   - If you can issue an invoice, submit it as expense to [Open Collective Engineering](https://opencollective.com/engineering).
+   - If you can't issue an invoice, ask to be rewarded with an Open Collective Gift Card.
+
+### Workflow for Core Contributors
+
+1. Tag issues with the "bounty" label and another tag with the bounty amount (ie: "100\$")
+
+1. Add a comment with basic explanations of the Bounty process and a link to BOUNTY.md
```
Comment (Betree) [inline BOUNTY.md]: It would be nice to provide a template for the comment to add on the issue

File: BOUNTY.md (hunk 1 above)
Comment (znarf) [reply BOUNTY.md]: That was integrated.

File: BOUNTY.md (hunk 2 above)
Comment (znarf) [reply BOUNTY.md]: That was integrated.

**Pull request merged**